tests/python/quantization/test_quantization.py

```diff
@@ -178,7 +178,7 @@ def check_requantize_with_symbol(shape, min_calib_range=None, max_calib_range=No
                                                                           max_range.asscalar(),
                                                                           min_calib_range=min_calib_range,
                                                                           max_calib_range=max_calib_range)
-        assert_almost_equal(qdata_int8.asnumpy(), qdata_int8_np)
+        assert_almost_equal(qdata_int8.asnumpy(), qdata_int8_np, atol = 1)
```

Comment:
Why the change here?

Reply:
Flaky test test_quantization_mkldnn.test_requantize_int32_to_int8 it failed on 1 of the builds. So I retriggered it. But also added that

Reply:
Reference https://github.com/apache/incubator-mxnet/pull/16709